generator/internal/dart/templates/pubspec.yaml.mustache

```diff
@@ -0,0 +1,21 @@
+{{!
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+}}
+name: {{Codec.DartPackage}}
```

Comment:
nit: can we *generate* the copyright boilerplate for these files?  If not, maybe add a mustache comment explaining why.

Reply:
yes, I was asking about adding it to the generated file.  We can do it later, but we have been taking the (~~extreme?~~ very conservative?) view that anything that looks like code (even generated code) needs the boilerplate.

Reply:
Gotcha! Done.